ajna/management/commands/test.py

```
@@ -0,0 +1,38 @@
+import importlib
```

Ruff (0.8.2)

`importlib` imported but unused

Remove unused import: `importlib`

(F401)